README.md

```diff
@@ -139,12 +139,16 @@ Install SQLMesh through [pypi](https://pypi.org/project/sqlmesh/) by running:
 ```bash
 mkdir sqlmesh-example
 cd sqlmesh-example
-python -m venv .env
-source .env/bin/activate
+python -m venv .venv
```

Comment:
should we change this to use UV to match cloud?

Reply:
Literally clicked merge when you commented. I recommend no given `uv` is still a new player and may be a intimidating to first-timers on our page. I'm okay with it being in tcloud docs as that requires a live explainer from someone.